app/src/main/java/com/codepath/apps/critterfinder/activities/PetDetailsActivity.java

```diff
@@ -7,6 +7,7 @@
 import android.support.v7.widget.Toolbar;
 
 import com.codepath.apps.critterfinder.R;
+import com.codepath.apps.critterfinder.adapters.PetFavoritesAdapter;
```

Comment:
I think this import is unused. The pet details activity doesn't communicate with the pet favorites adapter. I bet you can remove this line and it would still compile.